blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -79,11 +92,11 @@ export default async function decorate(block) {
                 </div>
                 <div class="kpis-wrapper">
                     <span class="h3">KPIs to Measure Success</span>
-                    ${kpis}
+                    ${buildKPIList(program).outerHTML}
```

Comment:
@TyroneAEM Is there any reason why `const kpis = buildKPIList(program).outerHTML;` is removed and directly calling `buildKPIList(program).outerHTML` in the HTML code?

Reply:
All the const variables have been restored, they removed when the code was restructured.